src/Pages/Nodes.razor

```diff
@@ -326,6 +370,21 @@
         await GetData();
 
     }
+    
+    private async Task StopJob(Node node)
+    {
+        try
+        {
+            IScheduler scheduler = await SchedulerFactory.GetScheduler();
+
+            await scheduler.UnscheduleJob(new TriggerKey(node.JobId));
```

Comment:
does this only uneschedule the job or also deletes the job from the database?

Reply:
just uneschedule right, make sense to delete from db.